parsec/logging.py

```diff
@@ -92,7 +101,8 @@ def _structlog_to_sentry_processor(logger, method_name: str, event: dict) -> dic
 
     if level in _SENTRY_EVENT_LEVELS:
         # Cook the exception as a (class, exc, traceback) tuple
-        v = data.pop("exc_info", None)
+        v: Any = data.pop("exc_info", None)
+        exc_info: Any
```

Comment:
`exc_info` seems to be `ExcInfo | None`
```python
ExcInfo: TypeAlias = tuple[type[BaseException], BaseException, TracebackType]
```
Careful though, `sys.exc_info()` returns an `OptExcInfo`, defined as:
```python
OptExcInfo: TypeAlias = Union[ExcInfo, tuple[None, None, None]]
```

Reply:
Done.